src/test/resources/hudson/plugins/warnings/parser/gnuMakeGcc.txt

```diff
@@ -34,7 +34,7 @@ warnings.cc:14:   instantiated from here
 warnings.cc:6: warning: passing 'Test' chooses 'int' over 'unsigned int'
 warnings.cc:6: warning:   in call to 'std::basic_ostream<_CharT, _Traits>& std::basic_ostream<_CharT, _Traits>::operator<<(int) [with _CharT = char, _Traits = std::char_traits<char>]'
 
-make[2]: Entering directory '/dir1/dir2/dir3'
```

Comment:
Wouldn't it make more sense to leave the old file and tests untouched and create a new ones that show the new feature only?

Reply:
I guess. Fixed in new commit.